orc8r/cloud/go/services/orchestrator/obsidian/models/defaults.go

```diff
@@ -44,6 +44,12 @@ func NewDefaultSentryConfig() *NetworkSentryConfig {
 	}
 }
 
+func NewDefaultStateConfig() *StateConfig {
```

Comment:
new unit test here?

Reply:
i think these are just helper functions for unit tests, looking at code usages i don't see it used anywhere outside of tests

Reply:
oh interesting. I see it's used across packages; probably belongs in an `internal/testutil` or `internal/test/fixtures` but fine for now :)

Reply:
yeah lets save that for a refactor, want to limit scope of this pr to handling the sync interval change